tests/testData/annotation/call_input_checking.wdl

```diff
@@ -0,0 +1,236 @@
+version 1.0
```

Comment:
If we do this check in draft-2, we should have a draft-2 version of this test file.